components/notifications.js

```diff
@@ -464,7 +470,7 @@ function Invoicification ({ n: { invoice, sortTime } }) {
               if (disableRetry) return
               setDisableRetry(true)
               try {
-                const { error } = await retry({ variables: { invoiceId: parseInt(invoiceId) } })
+                const { error } = await retry({ variables: { invoiceId: parseInt(invoiceId), resetAttempts: true } })
```

Comment:
Does this `resetAttempts` depend on the other pr?

Reply:
Yes, it is a left over from the extraction. I will clean it up